ynr/apps/popolo/migrations/0009_move_extra_person_data_to_base.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2018-08-14 17:04
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def add_extra_fields_to_base(apps, schema_editor):
+    Person = apps.get_model("popolo", "Person")
+    PersonExtra = apps.get_model("candidates", "PersonExtra")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    Image = apps.get_model("images", "Image")
+
+    # import ipdb; ipdb.set_trace()
+
+    # First, delete any Membership objects with no extra
+    Person.objects.filter(extra=None).delete()
```

Comment:
Copy and paste error here? The comment doesn't match the statement.

Also, not saying this is _wrong_ (on my local DB every `Person` also had a `PersonExtra` so nothing was deleted), but do we expect this to match and records in production and if so do we definitely want to delete that data? Just want to understand/check.

Reply:
I'll update the comment, yes copy/paste error.

As for if I expect any data to exist: no, I don't. On Prod `Person.objects.filter(extra=None)` is empty. I've added this here though because the constraint isn't enforced at the DB level. This means that there _could_ be some `PersonExtra` models floating about in other installs / in future. They will only be created by a bug (the `/person/[pid]` URL doesn't work without the paired model, loads of other stuff breaks), but I don't want that bug to break the migration. This is just a precaution.